documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -82,22 +82,33 @@ You can migrate virtual machines (VMs) to {virt} by using the {project-short} we
 [id="adding-providers"]
 === Adding providers
 
-You can add providers by using the {project-short} web console.
+You can add source providers and destination providers by using the {project-short} web console.
```

Comment:
Please add the text in bold:
You can add source providers and destination providers **for the virtual machines migration** by using the {project-short} web console.

Reply:
Done.